collection-scripts/gather_mr

```diff
@@ -2,6 +2,10 @@
 # shellcheck disable=SC1091
 source common.sh
 resources=("modelregistries")
+# depedent resources ossm
+resources+=("smcp" "smm" "smmr" "authorizationpolicies")
```

Comment:
```suggestion
resources+=("smcp" "smm" "smmr" "authorizationpolicies" "destinationrules" "gateways" "virtualservices")
```
Add additional servicemesh resources for model registry

Reply:
this is the most important part i would like component teams for review